scipy/stats/_multivariate.py

```diff
@@ -3870,7 +3871,8 @@ class random_correlation_gen(multi_rv_generic):
     Parameters
     ----------
     eigs : 1d ndarray
-        Eigenvalues of correlation matrix
+        Eigenvalues of correlation matrix. All eigenvalues need to be non-negative and
+        need to sum up to the dimension.
```

Comment:
Perhaps slightly clearer?
```suggestion
        need to sum to the number of eigenvalues.
```

Reply:
Yes, I'd say that "dimension" is not correct (here or most places it's used in `_multivariate.py`). "dimensionality" is often what is meant, but "number of eigenvalues" is even more clear here.